zerver/lib/types.py

```diff
@@ -72,3 +72,7 @@ class SAMLIdPConfigDict(TypedDict, total=False):
     extra_attrs: List[str]
     x509cert: str
     x509cert_path: str
+
+
+class NoValueSent:
+    pass
```

Comment:
This should have a docstrings explaining how it's a solution for parameters that want to distinguish an explicit `null`.

Reply:
Done.

Reply:
I did a further edit pass on this.